keps/sig-storage/1698-generic-ephemeral-volumes/README.md

```diff
@@ -441,7 +440,7 @@ version will prevent pods from starting.
 
   Yes, unit tests for the apiserver, kube-controller-manager and kubelet cover scenarios
   where the feature is disabled or enabled. Tests for transitions
-  between these states will be added before beta.
+  between these states [were added](https://github.com/kubernetes/kubernetes/pull/99446) before beta.
```

Comment:
Can't comment on the related lines, but:
L424 - has that been clearly documented? If not, can we ensure it is documented?

Reply:
I think it was covered sufficiently, both when announcing the feature and the current documentation:
- https://kubernetes.io/blog/2020/09/01/ephemeral-volumes-with-storage-capacity-tracking/#generic-ephemeral-volumes-1
- https://kubernetes.io/docs/concepts/storage/ephemeral-volumes/#security

/hold cancel

Reply:
The second link satisfied me - thanks!